front/src/applications/operationalStudies/views/Scenario/components/Timetable/TimetableBoardWrapper.tsx

```diff
@@ -114,6 +115,67 @@ const TimetableBoardWrapper = ({
     [selectedTimetableItemIds]
   );
 
+  const removeAndUnselectTrains = useCallback(
+    (timetableItemIds: TimetableItemId[]) => {
+      removeTimetableItems(timetableItemIds);
+      setSelectedTimetableItemIds([]);
+    },
+    [removeTimetableItems, setSelectedTimetableItemIds]
+  );
+
+  const ctrlC = useCallback(
+    (event: KeyboardEvent) => {
+      if (!event.repeat && event.ctrlKey && event.key === 'c') {
```

Comment:
I think we can use the `copy` event here instead too (together with `event.preventDefault()` to disable the built-in copy behavior).